### ISSUE TITLE
Exclude disabled entry

### ISSUE THREAD

**Issue by mark-chief** [Rias500/craft-scout#18]
I'd like to remove an entry from the Algolia index if it is later disabled, how would this be possible with the plugin? Thanks

**Comment by timkelty** [comment]
@mark-chief you need to do this yourself via your indexing rules.

Make sure your `criteria` contains `'status' => null,`, so that when an entry is saved as disabled, it will still index.

Then from the client (algolia) side, you need to pre-filter to only show `enabled`: https://www.algolia.com/doc/tutorials/search-ui/instant-search/filtering/add-default-filters/instantsearchjs/#filters-parameter

**Comment by timkelty** [comment]
@mark-chief I take it back, Scout already has deindexing logic for this, but you'll want to add this to your criteria:

```
                    'status' => null,
                    'enabledForSite' => false,
```

**Comment by Rias500** [comment]
Let me know if either of those options don't work, then it might be a bug. Closing for now

**Comment by mark-chief** [comment]
Hi, Sorry, I didn't actually get this to work in the end.

My criteria looks like this:

```
            'elementType' => \craft\elements\Entry::class,
            'criteria' => [
                'section' => 'websitePages',
                'status' => null,
                'enabledForSite' => false        
            ], 
```           
If I create an entry, Algolia syncs and I can see it in the Algolia Dashboard. if I delete the entry it remains in the Algolia dashboard and status remains live. thanks for your hard work.

**Comment by Rias500** [comment]
There was an issue with de-indexing of elements, this should now be fixed in 1.0.0!
